Wikipedia/Code/TalkPageHeaderView.swift

```diff
@@ -335,7 +335,21 @@ final class TalkPageHeaderView: SetupView {
         
         let theme = viewModel.theme
         
-        coffeeRollLabel.attributedText = coffeeRollText.byAttributingHTML(with: .callout, boldWeight: .semibold, matching: traitCollection, color: theme.colors.primaryText, linkColor: theme.colors.link, handlingLists: true, handlingSuperSubscripts: true, tagMapping: ["a": "b"])
+        var attributedText: NSMutableAttributedString = coffeeRollText.byAttributingHTML(with: .callout, boldWeight: .semibold, matching: traitCollection, color: theme.colors.primaryText, linkColor: theme.colors.link, handlingLists: true, handlingSuperSubscripts: true, tagMapping: ["a": "b"])
+        if attributedText.string.first == "\n" {
```

Comment:
I think this could move into the helper method too, just to simplify the call site. Treat it as a guard early in the method.